src/utils/secsToHMS.js

```diff
@@ -4,14 +4,17 @@
  * @return {String}             [hh:mm:ss]
  */
 function convertSecsToHMS(sec) {
+    if(sec == "0"){
```

Comment:
Is `duration` somewhere validated? We could use a `null` check too.

Reply:
oh yeap, make sense.